mula/scheduler/models/boefje.py

```diff
@@ -23,7 +17,8 @@ class BoefjeMeta(BaseModel):
     id: str
     boefje: Boefje
     input_ooi: Optional[str]
-    arguments: Dict[str, Any]
```

Comment:
This is actually a nice hint, unless the key type is changed?

Reply:
I changed it to resemble the same structure in boefjes. However I'm ok with reverting it, we should consider changing it to both to the correct form.